uktrade/data-hub-frontend
# Add reminder summary widget

## Description of change

This adds a reminder summary widget for personalised dashboard.

It requires user to have `reminder-summary` feature flag.

## Test instructions

Go to personalised dashboard, you should be able to see the widget on the left hand side, provided you have `reminder-summary` user feature flag set.

## Screenshots
### Before

![reminders-before](https://user-images.githubusercontent.com/5889630/177320466-4b887cf6-7b51-4899-9315-ff5f2ecda40e.png)

### After

![reminders-after](https://user-images.githubusercontent.com/5889630/177320477-254df1ca-52b9-4fba-9dec-5729eb5249ee.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
